packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js

```diff
@@ -16,162 +19,158 @@ angular
+          === $scope.model.password.confirmation
         && Hosting.constructor.isPasswordValid(
-          $scope.model.selected.password.value,
+          $scope.model.password.value,
           $scope.customPasswordConditions,
         );
 
-      function getAvailableVersion(type) {
-        $scope.model.capabilities.bdVersions = null;
-        $scope.model.selected.version = null;
-        HostingDatabase.getAvailableVersion($stateParams.productId, type).then((result) => {
-          $scope.model.capabilities.bdVersions = result.list;
-          $scope.model.selected.version = find(
-            $scope.model.capabilities.bdVersions,
-            elt => elt === result.default,
-          );
-        }); // no need error function
-      }
-
-      $scope.$watch('model.selected.type', () => {
-        if ($scope.model.selected.type) {
-          getAvailableVersion($scope.model.selected.type.toLowerCase());
-        }
-      });
-
       $scope.model = {
-        capabilities: null,
-        selected: {
-          type: null,
-          capabilitie: null,
-          password: {
-            value: null,
-            confirmation: null,
-          },
-          quota: null,
-          user: '',
-        },
-        types: {
-          EXTRA_SQL_PERSO: 'EXTRA_SQL_PERSO',
+        capability: null,
+        type: null,
+        version: null,
+        quota: null,
+        user: null,
+        password: {
+          value: null,
+          confirmation: null,
         },
-        maxUserLength: 6,
       };
 
       $scope.customPasswordConditions = {
         max: 30,
       };
 
-      $scope.load = () => {
-        $q.all({
-          capabilities: HostingDatabase.getCreationCapabilities($stateParams.productId),
-          availableDatabasesType: HostingDatabase.getDatabaseAvailableType($stateParams.productId),
+      $scope.load = () => OvhApiHostingWeb.v6()
+        .getDatabaseCreationCapabilities({
+          serviceName: $stateParams.productId,
         })
-          .then((resp) => {
-            $scope.model.capabilities = resp.capabilities;
-            $scope.model.maxUserLength = MAX_USER_LENGTH - resp.capabilities.primaryLogin.length;
-
-            if (resp.capabilities.availableDatabases.length === 1) {
-              [
-                $scope.model.selected.capabilitie,
-              ] = resp.capabilities.availableDatabases;
-            }
-
-            $scope.availableDatabasesType = resp.availableDatabasesType;
-            $scope.model.selected.type = 'MYSQL';
-          })
-          .catch((err) => {
-            Alerter.alertFromSWS(
-              $translate.instant('hosting_tab_DATABASES_configuration_create_step1_loading_error'),
-              get(err, 'data', err),
-              $scope.alerts.main,
-            );
+        .$promise
+        .then((capabilities) => {
+          $scope.capabilities = capabilities;
+
+          map($scope.capabilities, (capa) => {
+            const capability = capa;
```

Comment:
You could return directly something like 
```suggestion
            ({
            ...capability, 
            snakeType: snakeCase(capability.type)
            })
```
Moreover `formattedType` or `snakeCasedType` would be better I think